kuma/banners/jinja2/banners/includes/developer-needs.html

```diff
@@ -0,0 +1,15 @@
+<div id="developer-needs" class="mdn-cta-content-container">
+    <h2 class="mdn-cta-title">
+        {{ _('MDN Developer Survey') }}
+    </h2>
+    <p class="mdn-cta-copy">
+        {{ _('We want to know learn more about the needs of developers') }}
+    </p>
+</div>
+<p class="mdn-cta-button-container">
+    <a
+        href="https://developer.mozilla.org/"
+        class="mdn-cta-button mdn-cta-external-link"
+        >{{ _('Complete the survey') }}</a
```

Comment:
When this text appears on the button, to my eye it seems too small/thin compared to the size of the button (just a thought).